lib/routes/routePUT.js

```diff
@@ -38,6 +39,8 @@ export default function routePUT(request, response, log, statsClient) {
         request.post = '';
         const post = [];
         let postLength = 0;
+        // Send continue message for streaming V4 requests, if applicable.
+        writeContinue(request, response);
```

Comment:
If the `writeContinue` method is not called at this point, the check for `MAX_POST_LENGTH` will never be made since the body will not be sent.

https://github.com/scality/S3/blob/6a3115b9ae52238b77cfe5dbfac0a8e393336c2f/lib/routes/routePUT.js#L54